src/app/editor.cljs

```diff
@@ -63,9 +63,10 @@
                                                              [(sci/extension
                                                                {:modifier "Alt",
                                                                 :on-result
-                                                                (fn [result]
+                                                                (fn [code result]
+                                                                  (reset! source (str code))
```

Comment:
I don't think this is a clean solution, but (maybe) because `Editor`'s state is detached from reagent lifecycle, we need this `reset!` call to sync state of the `Editor` instance and our `editor` component.